webui/react/src/components/kit/LineChart.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useMemo, useState } from 'react';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { FixedSizeGrid } from 'react-window';
+import { AlignedData } from 'uplot';
+
+import SelectFilter from 'components/SelectFilter';
+import { SyncProvider } from 'components/UPlot/SyncableBounds';
+import UPlotChart, { Options } from 'components/UPlot/UPlotChart';
+import { tooltipsPlugin } from 'components/UPlot/UPlotChart/tooltipsPlugin';
+import { glasbeyColor } from 'shared/utils/color';
+import { Scale } from 'types';
+
+import css from './LineChart.module.scss';
+
+interface Serie {
+  color?: string;
+  data: number[][];
+  name?: string;
+}
+
+interface Props {
+  focusedSeries?: number;
+  height?: number;
+  onSeriesHover?: (seriesIdx: number | null) => void;
```

Comment:
This feels like more arguments than we actually need. It's not clear to me when we need most of these callbacks.

We want to avoid fixed width and standardize whether metrics are displayed, whether legends are displayed, etc.

Which of these are actually necessary

Reply:
✅ I can remove width from chart since that was needed only for `<AutoSizer>` -> `<FixedSizeGrid>`
I'll re-read the ticket re: callbacks or we can err on not having so many.  If the component can add/select/remove metrics on its own then we would need callbacks.

Reply:
I think that's only the case if you want to make it a controlled component, no? If the chart manages its own state you shouldn't need any callbacks.

Reply:
ok I drop the callbacks in this commit.  I think we have a component in multi-trial search where we currently have with chart interactions limit which trials show up in a table, but I might just remembering the parallel coordinates plot